src/main/java/frc/robot/subsystems/SimulationSubsystem.java

```diff
@@ -46,6 +50,19 @@ public void simulationPeriodic() {
     //Update simYaw distance
     simYaw += chassisRotationSpeed *  Robot.kDefaultPeriod;
 
+    //Update simPitch
+    swerveX = swerveSubsystem.getPose().getX();
+    
+    if (swerveX >= 0 && swerveX <= FieldConstants.Community.chargingStationLength) {
+      simPitch = Units.degreesToRadians((swerveX * ((FieldConstants.Community.chargingStationAngle * 2) / FieldConstants.Community.chargingStationLength)) - FieldConstants.Community.chargingStationAngle);
+    }
+    else {
+      simPitch = 0;
+    }
+
+    SimDouble pitch = new SimDouble(SimDeviceDataJNI.getSimValueHandle(navXSim, "Pitch"));
+    pitch.set(simPitch);
+
     //Updating Sim NavX
     SimDouble angle = new SimDouble(SimDeviceDataJNI.getSimValueHandle(navXSim, "Yaw"));
```

Comment:
Move the SimDouble creations to the top of the class with the private variables (and make them private as well)

Reply:
Done